src/pages/integrations/analytics/custom-events/index.md

```diff
@@ -0,0 +1,110 @@
+
+PWA Studio provides a [sample extension](https://github.com/magento/pwa-studio/tree/develop/packages/extensions/experience-platform-connector/) that shows how to add a custom event.
+
+In the sample extension, the `SEARCH_REPSONSE` event is being tracked. This event is fired when a search result is returned from the Adobe Experience Manager (AEM), with either results or a `EMPTY_RESULT` response.
+
+These files can be used as a starting point for a custom event extension.
+
+## searchResponseReceived.js
+
+This file sets the payload and the search results context. The custom event name is also defined in this file.
+
+```javascript
+const canHandle = event => event.type === 'SEARCH_RESPONSE';
+
+...
+
+const { payload } = event;
+
+    const {
+        categories,
+        facets,
+        page,
+        perPage,
+        products,
+        searchRequestId,
+        searchUnitId,
+        suggestions
+    } = payload;
+
+    const searchResultsContext = {
+        units: [
+            {
+                categories,
+                facets,
+                page,
+                perPage,
+                products,
+                searchRequestId,
+                searchUnitId,
+                suggestions
+            }
+        ]
+    };
+
+    sdk.context.setSearchResults(searchResultsContext);
+    sdk.publish.searchResponseReceived(searchUnitId, searchResultsContext);
+};
+```
+
+## wrapUseAutocomplete.js
+
+In `wrapUseAutocomplete.js` the search result is received and the payload defined and dispatched. This take place in the `useEffect` function:
+
+```javascript
+useEffect(() => {
+            if (
+                messageType === 'RESULT_SUMMARY' ||
+                messageType === 'EMPTY_RESULT'
+            ) {
+                dispatch({
+                    type: 'SEARCH_RESPONSE',
+                    payload: {
+                        categories: categories || [],
+                        facets: [],
+                        page: 1,
+                        perPage: 3, // Same value used in GQL query
+                        products: displayResult || [],
+                        searchRequestId: value,
+                        searchUnitId: 'search-bar',
+                        suggestions: displayResult || []
+                    }
+                });
+            }
+```
+
+## intercept.js
+
+Edit copy of `intercept.js` to pull in the custom extension by name. In this case, the `SearchBar` wrapper:
+
+```javascript
+    talons.tap(({ App, Header, SearchBar }) => {
+        App.useApp.wrapWith('@magento/experience-platform-connector');
+        Header.useAccountMenu.wrapWith(
+            '@magento/experience-platform-connector/src/wrappers/wrapUseAccountMenu'
+        );
+        SearchBar.useAutocomplete.wrapWith(
+            '@magento/experience-platform-connector/src/wrappers/wrapUseAutocomplete'
+        );
+```
+
+## Import a custom extension
```

Comment:
Not sure this is needed. Most developers should already know how to set up a new package but if you really want to start from scratch, I would refer to the section in this tutorial on [how to initialize a project](https://developer.adobe.com/commerce/pwa-studio/tutorials/targets/modify-talon-results/#initialize-the-project).

I would replace this particular section with a testing locally section that looks like [this](https://developer.adobe.com/commerce/pwa-studio/tutorials/targets/modify-talon-results/#test-on-a-local-instance)